Kickstarter-iOS/Views/Storyboards/CreditCardCell.xib

```diff
@@ -55,13 +56,13 @@
                     </view>
                 </subviews>
                 <constraints>
-                    <constraint firstAttribute="trailing" secondItem="DRF-8a-Nrl" secondAttribute="trailing" constant="6" id="7xh-nj-DfR"/>
+                    <constraint firstAttribute="trailing" secondItem="DRF-8a-Nrl" secondAttribute="trailing" constant="12" id="7xh-nj-DfR"/>
                     <constraint firstAttribute="trailing" secondItem="qhp-KL-uM9" secondAttribute="trailing" id="IT1-1x-kYH"/>
                     <constraint firstItem="qhp-KL-uM9" firstAttribute="top" secondItem="DRF-8a-Nrl" secondAttribute="bottom" constant="6" id="LlQ-rl-2uz"/>
                     <constraint firstAttribute="bottom" secondItem="qhp-KL-uM9" secondAttribute="bottom" constant="-0.5" id="MKp-2I-ihI"/>
                     <constraint firstItem="DRF-8a-Nrl" firstAttribute="top" secondItem="eaL-oG-LUu" secondAttribute="top" constant="6" id="TkB-SH-Xh3"/>
                     <constraint firstItem="qhp-KL-uM9" firstAttribute="leading" secondItem="eaL-oG-LUu" secondAttribute="leading" id="UzA-dc-x3a"/>
-                    <constraint firstItem="DRF-8a-Nrl" firstAttribute="leading" secondItem="eaL-oG-LUu" secondAttribute="leading" constant="6" id="aUd-nJ-OF6"/>
+                    <constraint firstItem="DRF-8a-Nrl" firstAttribute="leading" secondItem="eaL-oG-LUu" secondAttribute="leading" constant="12" id="aUd-nJ-OF6"/>
```

Comment:
This is the left padding of the stack view

Reply:
@dusi I wonder if it might make more sense to pin the edges of the `stackView` to the `layoutMargins` of the `contentView`, and then programmatically set the `layoutMargins` to be the correct padding values?

Reply:
We can use `layoutMargins` (on iOS 10) and `directionalLayoutMargins` on `iOS 11+` (see the docs https://developer.apple.com/documentation/uikit/uiview/1622566-layoutmargins.

Is there a specific reason we'd prefer to use `layoutMargins` as oppose to auto layout? If it's only so that we can expose this in code I'd be probably more in favour of going with this solution since the rest of this UI is still defined in the `CreditCardCell.xib`